cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -811,10 +811,10 @@ var Plugin = new Class({
      * Destroys the current plugin instance removing only the DOM listeners
      *
      * @method destroy
-     * @param {Boolean} mustCleanup - if true it will remove also the plugin UI components from the DOM
+     * @param {Boolean} options.mustCleanup - if true it will remove also the plugin UI components from the DOM
      * @returns {void}
      */
-    destroy(mustCleanup) {
+    destroy({ mustCleanup }) {
```

Comment:
should be`{ mustCleanup = false } = {}`, the unit tests fail when object isn't passed

Reply:
ah right i will fix it asap (probably tomorrow) sorry.. I am overloaded of things to do

Reply:
ok now it should be fine and the tests are green